config/initializers/time_formats.rb

```diff
@@ -0,0 +1,2 @@
+# Define a custom date format by name
+Time::DATE_FORMATS[:merchants] = "%A, %B %-d, %Y"
```

Comment:
This is neat to see, I am curious, I think other places we have used a method to format this (I used it in invoice).  Is this what allows the `to_formatted_s(:merchants)` in the merchants show view?

If so might be good to use and apply elsewhere.

Reply:
Yeah this is what I commented on in your PR. Makes sense to abstract it outside the model itself so many views can use it